apps/web/scripts/docker/read-secrets.sh

```
@@ -2,15 +2,44 @@
+  export ENCRYPTION_KEY
 else
   echo "ENCRYPTION_KEY secret not found. Build may fail if this is required."
 fi
 
+if [ -f "/run/secrets/sentry_auth_token" ]; then
+  # Only upload sourcemaps on amd64 platform to avoid duplicate uploads
+  # Sourcemaps are platform-agnostic, so we only need to upload once
+  # TARGETARCH is automatically set by Docker during multi-platform builds
+  if [ "${TARGETARCH:-}" = "amd64" ]; then
+    IFS= read -r SENTRY_AUTH_TOKEN < /run/secrets/sentry_auth_token || true
+    SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN%$'\n'}
```

Shellcheck (0.10.0)

[warning] In POSIX sh, $'..' is undefined.

(SC3003)